### PR TITLE
Home page icons view

### PR DESCRIPTION
**Pull request by jiayinjx**
1. make the app can be delete from the home-page icon view by the cross symbol
2. add description to each app which will let the user know which app they are at before launching.

### CODE REVIEW

File: angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/widget-card.html [hunk 1]
```diff
@@ -9,14 +9,14 @@
...
+      <i title="Remove" class="fa fa-times portlet-options" ng-click="widgetCtrl.removePortlet(portlet.nodeId, portlet.title)"><a aria-label="Remove this app" href="#"></a></i>
     </div>
     
     <div class="widget-title">
-      <h4>{{ ::portlet.title }}</h4>
+      <h4 id="appTitle{{::portlet.title}}" aria-labelledby="appTitle{{::portlet.title}}" tabindex="0">{{ ::portlet.title }}</h4>
     </div>
   </div>
-  
+  <sub hidden="" id="goToApps">go to</sub>
```
Comment (timlevett) [inline angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/widget-card.html]: Again, the id must be unique across the whole document. The `widget-card` partial is used many times.  Also if you want to hide for everything but a screen reader just use `class='sr-only'`. Its a bootstrap class. Reason for the `<sub>` tag?

Comment (jiayinjx) [inline angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/widget-card.html]: I didn't see any id like goToApps...... so, change to id="goToApp{{::portlet.title}" work?

Comment (timlevett) [inline angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/widget-card.html]: I see what you mean, so you are concatenating the fields together to give a better aria-label in the end. I would still suggest the `id="goToApp{{::portlet.title}"`.

Comment (jiayinjx) [inline angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/widget-card.html]: Yes, 
widgecard
so is that ok if I just change the id and leave <sub> tag there like this?

Comment (timlevett) [inline angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/widget-card.html]: Yes, that sounds good

**Pull request merged**